documentation/modules/proc-accessing-kafka-using-nodeports.adoc

```diff
@@ -35,36 +35,38 @@ spec:
         authentication:
           type: tls
         configuration:
-          brokerCertChainAndKey: <1>
+          brokerCertChainAndKey: <2>
             secretName: my-secret
             certificate: my-certificate.crt
             key: my-key.key
-          preferredAddressType: InternalDNS <2>
+          preferredAddressType: InternalDNS <3>
     # ...
   zookeeper:
     # ...
 ----
-<1> Optional configuration for a xref:kafka-listener-certificates-str[Kafka listener certificate] managed by an external Certificate Authority. The `brokerCertChainAndKey` property specifies a `Secret` that holds a server certificate and a private key. Kafka listener certificates can also be configured for TLS listeners.
-<2> Optional configuration to xref:con-kafka-broker-external-listeners-nodeports-{context}[specify a preference for the first address type used by Strimzi as the node address].
+<2> Optional configuration for a xref:kafka-listener-certificates-str[Kafka listener certificate] managed by an external Certificate Authority. The `brokerCertChainAndKey` property specifies a `Secret` that holds a server certificate and a private key. Kafka listener certificates can also be configured for TLS listeners.
+<3> Optional configuration to xref:con-kafka-broker-external-listeners-nodeports-{context}[specify a preference for the first address type used by Strimzi as the node address].
 
 . Create or update the resource.
 +
 [source,shell,subs=+quotes]
 kubectl apply -f _KAFKA-CONFIG-FILE_
++
+`NodePort` type services are created for each Kafka broker, as well as an external _bootstrap service_.
+The bootstrap service routes external traffic to the Kafka brokers.
+Node addresses used for connection are propagated to the `status` of each service.
 
-. Find the port number of the bootstrap service.
+. Retrieve the connection details to access the Kafka cluster.
 +
 [source,shell,subs=+quotes]
-kubectl get service _KAFKA-CLUSTER-NAME_-kafka-external-bootstrap -o=jsonpath='{.spec.ports[0].nodePort}{"\n"}'
+kubectl get kafka _KAFKA-CLUSTER-NAME_ -o=jsonpath='{.status.listeners[?(@.type=="external")].bootstrapServers}{"\n"}'
 +
-The port is used in the Kafka _bootstrap_ address.
-
-. Find the address of the Kubernetes node.
+You need:
```

Comment:
From this point on the rest of the `Retrieve the connection details to access the Kafka cluster.` section should be removed.

Reply:
Can we confirm/verify what the reader needs to pick up from the fetch? 

> Use the address and port number of one of the kafka broker nodes (_NODE-ADDRESS:NODE-PORT_) in your Kafka client as the _bootstrap address_ to connect to the Kafka cluster.

Reply:
When the user runs this command: `kubectl get kafka _KAFKA-CLUSTER-NAME_ -o=jsonpath='{.status.listeners[?(@.type=="external")].bootstrapServers}{"\n"}'` ... it will give back the bootstrap address ... so the `_NODE-ADDRESS:NODE-PORT_`. That is why the user doesn't need anything more.

Reply:
Removed